assignment08/index.php

```diff
@@ -0,0 +1,34 @@
+<?php
+    include "includes/header.php";
+
+    $first_name = 'Dissa';
+    echo $first_name;
+    echo "<br>";
+    $last_name = 'Kondo';
+    echo $last_name;
+    echo "<br>";
+    $my_age = 36;
+    echo $my_age;
+    echo "<br>";
+    $my_weight = 170;
+    echo $my_weight;
```

Comment:
good variable declarations! You should not have echo statements here (unless you were just using them for debugging).